docs/reference/data-streams/tsds.asciidoc

```diff
@@ -147,6 +147,13 @@ example, a count of errors or completed tasks.
 Only numeric and `aggregate_metric_double` fields support the `counter` metric
 type.
 
+[[counter-field-limitation]]
+.Counter field limitation
+****
+ Note that counter fields do come with a limitation in aggregations. Only the following aggregations are supported with counter field:
+ `rate`, `histogram`, `range`, `min`, `max`, `top_metrics` and `variable_width_histogram`.
+****
```

Comment:
```suggestion
NOTE: Counter fields do come with a limitation in aggregations. Only the following aggregations are supported with the `counter` field: `rate`, `histogram`, `range`, `min`, `max`, `top_metrics` and `variable_width_histogram`.
```

Thanks for moving section. I'd also change the format so that the output looks like this:
![Screenshot 2023-04-11 at 11 46 50 AM](https://user-images.githubusercontent.com/41695641/231217614-bfc97dcb-d539-47a4-a042-7c6e731220a5.png)

Rather than like this: 
![Screenshot 2023-04-11 at 11 45 07 AM](https://user-images.githubusercontent.com/41695641/231217676-191e4127-93fa-4667-b097-a69427287415.png)
: